### DISCUSSION TITLE
Add Notion Template Support | 支持配置 Notion 模版

### DISCUSSION THREAD

**Discussion by iamStephenFang** [yihong0618/running_page#614] [General]
I wish the repo can support the one-click deployment of Github Actions to Notion templates, similar to this design：https://github.com/malinkang/weread2notion-pro

希望能够支持配置Github Actions一键部署到Notion的模板，类似这个设计：https://github.com/malinkang/weread2notion-pro

**Comment by yihong0618** [comment]
of course, 但是因为我不用 Notion 可能短时间内不会做，PR welcome, 设计的话没问题。
